### ISSUE TITLE
Add Client Registration Page link

### ISSUE THREAD

**Issue by teresas** [globusonline/docs.globus.org#14]
(was JIRA-GWEB-437)

needed to remove since page wasn't up yet.
add link:https://auth.globus.org/p/projects[Client Registration Page] back to auth api reference page (overview section)

**Comment by teresas** [comment]
@bjmc Should this be added back yet?

**Comment by jaswilli** [comment]
I don't think so--those pages aren't quite ready for public consumption yet. Also, I think they might be moving out of the `/p/` (private) namespace into the `/web/` (public) namespace.
